### PR TITLE
refactoring

### PR DESCRIPTION
**Pull request by bsach64**

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -50,52 +52,56 @@ func Listen(s SFTPServer) {
 
 	listener, err := net.Listen("tcp", addr)
 	if err != nil {
-		log.Fatalf("Failed to listen on 2022: %v", err)
+		log.Printf("Failed to listen on 2022: %v\n", err)
```
Comment (Unic-X) [inline server/server.go:55]: isko change krke "Failed to listen on port %i, server.port krde

File: server/server_test.go [hunk 2]
```diff
@@ -12,17 +12,22 @@ func TestKey(t *testing.T) {
 	s := New("0.0.0.1", "../private/id_rsa", 2022)
 	_, err := os.Stat(s.IdRsa)
 	if os.IsNotExist(err) {
-		fmt.Println("File does not exist:", s.IdRsa)
+		log.Println("File does not exist:", s.IdRsa)
 		t.Fatalf("ID_RSA file does not exist")
 	} else {
-		fmt.Println("File exists:", s.IdRsa)
+		log.Println("File exists:", s.IdRsa)
```
Comment (Unic-X) [inline server/server_test.go:18]: Fatal krde kya isko? println ki jagah?

**Review by Unic-X — CHANGES_REQUESTED** [review]

File: server/server_test.go (hunk 2 above)
Comment (Unic-X) [reply server/server_test.go:18]: sorry upar waale ko

File: server/server.go (hunk 1 above)
Comment (bsach64) [reply server/server.go]: Done!

**Review by Unic-X — APPROVED** [review]
LGTM!

**Pull request merged**